e2e_playwright/st_session_state_test.py

```diff
@@ -29,3 +29,58 @@ def test_checking_checkbox_unchecks_other(app: Page):
 
     expect(first_checkbox.locator("input")).to_have_attribute("aria-checked", "false")
     expect(second_checkbox.locator("input")).to_have_attribute("aria-checked", "true")
+
+
+def test_has_correct_starting_values(app: Page):
+    expect(app.get_by_text("item_counter: 0")).to_have_count(1)
+    expect(app.get_by_text("attr_counter: 0")).to_have_count(1)
+    expect(app.get_by_text("len(st.session_state): 5")).to_have_count(1)
+    expect(app.get_by_test_id("stJson")).to_be_visible()
+
+
+def test_can_do_CRUD_for_session_state_items(app: Page):
+    expect(app.get_by_text("item_counter: 0")).to_have_count(1)
+    expect(app.get_by_text("attr_counter: 0")).to_have_count(1)
+
+    button_element = app.get_by_test_id("stButton").locator("button").first
```

Comment:
nit: I think you can replace all button clicks in this file via the `click_button` method from `app_utils`. This has a coupe of benefit, e.g. it makes it easier to fix button clicks in all test files of something changes with the button internals